tools/iceoryx_build_test.sh

```diff
@@ -95,6 +97,11 @@ while (( "$#" )); do
         TEST_FLAG="ON"
         shift 1
         ;;
+    "test-add-user")
+        TEST_ADD_USER="ON"
+        echo "Warning: Tests are running with user accounts 'roudi_testX', please make sure that add_test_users.sh has run before."
+        shift 1
+        ;;
```

Comment:
This can easily be overlooked because there is a wall of output from the tests when the script is run. Could you maybe call the `add_test_users.sh` with a `check` parameter which aborts the script if not all users are available?